components/timeline/__tests__/image.test.ts

```diff
@@ -1,5 +1,5 @@
 import { imageDemoTest } from '../../../tests/shared/imageTest';
 
 describe('Timeline image', () => {
-  imageDemoTest('timeline', { skip: ['pending.tsx'] });
+  imageDemoTest('timeline');
```

Comment:
```suggestion
  imageDemoTest('timeline', { skip: ['pending.tsx'] });
```

有个 loading 图标，还得 skip 掉。